### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by arnetheduck**
* initial merge guide
* mark JSON-RPC as removed
* expand on binaries
* recommend `-j4` when building
* cleanups

### CODE REVIEW

File: docs/the_nimbus_book/src/eth1.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+# Run an execution client
 
-In order to be able to produce blocks and process incoming validator deposits from the execution layer, you'll need to run an execution client in together with the beacon node.
+In order to be able to produce blocks and process incoming validator deposits, you'll need to run an execution client in together with the beacon node.
 
-Nimbus has been tested all the major execution clients - see the [execution client comparison](https://ethereum.org/en/developers/docs/nodes-and-clients/#execution-clients) for more information.
+Nimbus has been tested all major execution clients - see the [execution client comparison](https://ethereum.org/en/developers/docs/nodes-and-clients/#execution-clients) for more information.
 
-By default, Nimbus uses WebSockets to communicate with the execution client, connecting to a server on the same machine using port 8546: http://localhost:8546.
+By default, Nimbus uses WebSockets to communicate with the execution client, connecting to http://localhost:8546. You can provide a different URL with the `--web3-url` parameter.
```
Comment (zah) [inline docs/the_nimbus_book/src/eth1.md]:
```suggestion
By default, Nimbus uses WebSockets to communicate with the execution client, connecting to ws://localhost:8546. You can provide a different URL with the `--web3-url` parameter.
```

File: docs/the_nimbus_book/src/eth1.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-> ⚠ You need to run your own execution client after The Merge - third-party services like Infura, Alchemy and Pocket will no longer be enough.
+> ⚠️ You need to run your own execution client after [the merge](./merge.md) - third-party services like Infura, Alchemy and Pocket will be sufficient.
```
Comment (zah) [inline docs/the_nimbus_book/src/eth1.md]:
```suggestion
> ⚠️ You need to run your own execution client after [the merge](./merge.md) - relying on third-party services such as Infura, Alchemy and Pocket will not be possible.
```

File: docs/the_nimbus_book/src/keep-updated.md [hunk 3]
```diff
@@ -8,28 +8,33 @@ To update to the latest version, either download the binary or compile the beaco
...
+```bash
+# Download the updated source code
 git pull && make update
 ```
 
 Followed by:
 
 ```
-make nimbus_beacon_node
+make -j4 nimbus_beacon_node
```
Comment (zah) [inline docs/the_nimbus_book/src/keep-updated.md:37]: Why 4 in particular?

Comment (arnetheduck) [reply docs/the_nimbus_book/src/keep-updated.md:37]: it provides "most" of the benefit already, is simple (no cross-platform cpu counting issues) and doesn't explode wrt memory usage on "typical" systems.

**Pull request merged**